### PR TITLE
Updated Hosts and Health diagnostics APIs documentations

### PR DESCRIPTION
**Pull request by aleksandrychev**

### CODE REVIEW

File: api/enterprise-api-ref/host.markdown [hunk 1]
```diff
@@ -295,6 +295,61 @@ HTTP 200 Ok
 }
 ```
 
+## Restore deleted hosts entry
+
+**URI:** https://hub.cfengine.com/api/hosts/restore-deleted/:host-id
+
+**Method:** POST
+
+Restore host entry in reporting database .
```
Comment (craigcomstock) [inline api/enterprise-api-ref/host.markdown:304]:
```suggestion
Restore host entry in reporting database.
```

File: api/enterprise-api-ref/host.markdown [hunk 2]
```diff
@@ -295,6 +295,61 @@ HTTP 200 Ok
...
+
+Note: to abe able to perform this action related RBAC rule should be enabled.
```
Comment (craigcomstock) [inline api/enterprise-api-ref/host.markdown:306]:
```suggestion
Note: to be able to perform this action related RBAC rule should be enabled.
```

Comment (craigcomstock) [reply api/enterprise-api-ref/host.markdown:306]: Do we want to add a reference link or actual name of RBAC rule either UI version and/or database format?

File: api/enterprise-api-ref/host.markdown [hunk 3]
```diff
@@ -295,6 +295,61 @@ HTTP 200 Ok
...
+
+## Permanently delete host entry
+
+**URI:** https://hub.cfengine.com/api/hosts/delete-permanently/:host-id
+
+**Method:** POST
+
+Permanently deletes host entry from the reporting database.
+
+Note: to abe able to perform this action related RBAC rule should be enabled.
```
Comment (craigcomstock) [inline api/enterprise-api-ref/host.markdown:334]:
```suggestion
Note: to be able to perform this action related RBAC rule should be enabled.
```

**Review by craigcomstock — CHANGES_REQUESTED** [review]

File: api/enterprise-api-ref/host.markdown (hunk 2 above)
Comment (nickanderson) [reply api/enterprise-api-ref/host.markdown:306]: I would guess `deletePermanently` and `restoreDeletedHosts`
https://github.com/cfengine/mission-portal/pull/2463/files#diff-00c4fcc2f1a37491edbfb456049471eebed9694282dd2f551010ab0b9877ab27R80

**Review by craigcomstock — APPROVED** [review]

**Pull request merged**